test/build_payment.js

```diff
@@ -7,6 +7,7 @@ describe('Ripple REST Client buildPayment', function() {
 
   before(function () {
     client = new Client({
+      api: 'http://localhost:5990/',
```

Comment:
We're testing agains the api.ripple.com because we want to catch breaks in future ripple-rest releases. In a future pull request, we'll have offline testing to make sure our internal (ripple-rest-client) logics work.